### PR TITLE
Add new functions to support quiet limit order failure

### PR DESCRIPTION
**Pull request by jarry-xiao**

### CODE REVIEW

File: typescript/phoenix-sdk/package.json [hunk 1]
```diff
@@ -18,16 +18,17 @@
     "@metaplex-foundation/rustbin": "^0.3.1",
     "@metaplex-foundation/solita": "^0.12.2",
     "@solana/spl-token": "^0.3.7",
-    "@types/node": "^18.11.13",
     "bn.js": "^5.2.1",
     "borsh": "^0.7.0",
     "bs58": "^5.0.0",
+    "fs": "^0.0.1-security",
```
Comment (eugene-chen) [inline typescript/phoenix-sdk/package.json:24]: what is this?

Comment (eugene-chen) [reply typescript/phoenix-sdk/package.json:24]: Also bump version

**Review by eugene-chen — APPROVED** [review]
why is there such a big change to yarn.lock

**Comment by jarry-xiao** [comment]
> why is there such a big change to yarn.lock

I'll revert it

**Pull request merged**